### ISSUE TITLE
GeoJSON property names are single characters

### ISSUE THREAD

**Issue by robinsummerhill** [spacesyntax/OpenMapping#4]
The property names of your GeoJSON output are single-character names (see below). If this is intentional (to reduce file size, for example) then it should be documented and a mapping from the single-character name to the full name should be provided.

```json
"properties{
  "id":578553,
  "m":53.7121960079831,
  "a":1098780,
  "b":81554,
  "c":3004,
  "o":"O13URF0MNRUAV",
  "r":"CENTRAL WAY",
  "d":"minor",
  "e":202,
  ...
}
```

**Comment by double-u-a** [comment]
Yes it was a minifier, but the minifier did not include the key mapping for some reason, will investigate to see how to ensure the mapping is included and documented in the next release.
